directory.json

```diff
@@ -191,5 +191,6 @@
   "verschwoerhaus": "https://verschwoerhaus.de/feed/spaceapi",
   "vspace.one": "https://vspace.one/spaceapi.json",
   "xHain": "https://x-hain.de/spaceapi-0.13.json",
-  "z-Labor Zwickau": "http://api.service.z-labor.space/spaceapi.json"
+  "z-Labor Zwickau": "http://api.service.z-labor.space/spaceapi.json",
+  "Port39": "https://port39.de/spaceapi.json"
```

Comment:
The list is sorted. Can you put it below `Pomerania Hackerspace`?

Reply:
sure - i will fix this... i just checked the list and it seemed not to be sorted, sry for this

Regarding the missing header i will try my best, but until now, it's only a managed webserver... we just started our space and services so it's not the best until now

Reply:
Is it a static JSON file? If you use a scripting language like PHP, you can set HTTP headers: https://www.php.net/manual/en/function.header.php

Reply:
it's just a static json file which is pushed by our door-Pi
i will check for Headers & v0.14 ^^

Reply:
If you can't do it for the moment, let us know, then we can merge as-is. The problem is that without the CORS headers, the endpoint will not be usable from web-based applications 🙂

Reply:
header should just be inserted ... at least the response in dev-tools includes the "Access-Control-Allow-Origin "*"" now, if this is sufficiant?